styles.css

```diff
@@ -0,0 +1,320 @@
+* {
+    font-family: 'Montserrat', sans-serif;
+  }
+  
+  sub {
+    font-size: 40px;
+  }
+  
+  h1 {
+    font-size: 80px;
+    margin: 0;
+    padding: 0;
+  }
+  
+  h2 {
+    font-size: 30px;
+  }
+  
+  h3 {
+    font-size: 20px;
+  }
+  
+  /* Main wrapper grid */
+  
+  .wrapper {
```

Comment:
Good use of descriptive class names. It makes it very clear which section the CSS is for.